### PR TITLE
Clarification styling

### LINKED ISSUE #77: Clarifying the programme builder steps
It needs to be clearer that users have to create a column/date before a session:

Harry's suggested draft:

screen shot 2017-03-15 at 11 00 14

### PR DESCRIPTION
**Pull request by naazy**
#77 clarification new session view
#68 validation when a user clicks on save			
fix #77 hiding track name when no track
Published Programme Message no longer distorts view #70

### Additions:
```
   .form__question-section--divided {
        border-bottom: solid 1px #c0c0c2;
      }
```

### Changes:
```
.prog-controls__dividing-section{
    display:inline-block;
    padding: 0.6875rem 1rem 0.625rem; /* 11px 16px 10px */
    border-left: solid 1px #c0c0c2;
    margin-left: 1rem;
  }

.prog-table__header--datetime {
    padding: 1rem;
    font-family: 'Raleway', sans-serif;
}
.prog-table__header {
    padding: 1rem 0.25rem;
    text-align: left;
}
```

### CODE REVIEW

File: src/NewColumnView.elm [hunk 1]
```diff
@@ -84,14 +97,14 @@ view model =
                         ]
                         [ text "Add New Column" ]
                     ]
-                , div [ class "prog-form--warning" ] [ text (newColumnWarning model) ]
+                , validationWarningDiv
                 , button
                     [ class "button button--secondary"
```
Comment (harrygfox) [inline src/NewColumnView.elm]: I know it's outside the diff and I'm sorry.
Please can we have these take the --wider modifier as well as the save button

File: src/NewSessionView.elm [hunk 2]
```diff
@@ -171,86 +168,114 @@ view context model =
...
+                    [ class "form__input form__input--time-min-prog-builder"
+                    , type_ "number"
+                    , value (toStringIgnore0 context.session.endTime.minute)
+                    , onInput UpdateNewSessionEndMinute
+                    , placeholder "00"
+                    ]
+                    []
+                , validationWarningDiv
+                , button
+                    [ class "button button--secondary"
```
Comment (harrygfox) [inline src/NewSessionView.elm]: Please can we have these take the --wider modifier as well as the save button

File: src/NewTrackView.elm [hunk 3]
```diff
@@ -96,14 +109,14 @@ view model =
                     , onClick AddNewTrack
                     ]
                     [ text "Add New Track" ]
-                , span [ class "prog-form--warning" ] [ text (newTrackWarning model) ]
+                , validationWarningDiv
                 , button
                     [ class "button button--secondary"
```
Comment (harrygfox) [inline src/NewTrackView.elm]: outside the diff - I'm sorry.
Please can we have these take the --wider modifier as well as the save button

**Comment by naazy** [comment]
Comments addressed @harrygfox

**Pull request merged**